src/odemis/gui/comp/stream.py

```diff
@@ -1651,7 +1651,7 @@ def _add_remove_btn(self):
     def _add_text_ctrl(self, name):
         """ Add a label control to the header panel """
         txt_ctrl = wx.TextCtrl(self, wx.ID_ANY, name, style=wx.TE_PROCESS_ENTER | wx.BORDER_NONE,
-                               validator=PatternValidator(r'[A-Za-z0-9_()-]+'))
+                               validator=PatternValidator(r'[A-Za-z0-9_()-]+'), size=(35, 35))
```

Comment:
You're only interested in increasing the vertical size, right? If so, I think that the first 35 (x) is a red herring. It should be possible to replace it by `-1`, which indicates to wxPython that the current/default value shouldn't be changed.